File: local alignment/shellScripts/echo.py
"""
    This script will take in a variable number of command line arguments
    and will print the the total number of arguments along with each
    one on a different line.

    NOTE: This script is written for python 2.7.

    Here is an example run of the program and the output it will produce
    
    $ python echo.py 1 2 3
    Total number of arguments:  3
    Argument  0  :  1
    Argument  1  :  2
    Argument  2  :  3
 """

import sys

num_params = len(sys.argv)-1

print "Total number of arguments:",num_params

for v in xrange(num_params):
    print "Argument",v,":",sys.argv[v+1]

File: HW3/shellScripts/Echo.java
/**
  This program will take in a variable number of command line arguments
  and will print the total number of arguments, followed by each one
  to the screen.  Here is an example of how to run it and what the
  output might look like:

  $ javac Echo.java
  $ java Echo 1stArg 2ndArg 3rdArg
    Total number of arguments: 3
    Argument 0 : 1stArg
    Argument 1 : 2ndArg
    Argument 2 : 3rdArg
 **/

public class Echo {

    public static void main(String[] args) 
    {
            int numArgs = args.length;
            
            System.out.println("Total number of arguments: " + numArgs);

            for (int i = 0; i < numArgs; i++)
            {
                    System.out.println("Argument " + i + " : " + args[i]);
            }
    
    }

}
